app/models/configuration.rb

```diff
@@ -30,11 +30,26 @@ def self.max_deleted_inline
   end
 
   # --------------------------------------------------------------- admin:paths
+  def self.paths_templates_methodologies
+    create_with(value: Rails.root.join('../../shared/templates/methodologies/').to_s)
+      .find_or_create_by(name: 'admin:paths:templates:methodologies').value
+  end
+
+  def self.paths_templates_notes
+    create_with(value: Rails.root.join('../../shared/templates/notes/').to_s)
+      .find_or_create_by(name: 'admin:paths:note_templates').value
+  end
+
   def self.paths_templates_plugins
     create_with(value: Rails.root.join('templates', 'plugins').to_s)
       .find_or_create_by(name: 'admin:paths:templates:plugins').value
   end
 
+  def self.paths_templates_projects
```

Comment:
Are we even using this directory for CE outside of the welcome task?

Reply:
No, I don't think so.
But it makes our Pro and CE codes converge. The `configuration` model now looks the same both in `Pro` and `CE`.